packages/types/src/lib/interfaces.ts

```diff
@@ -214,83 +214,44 @@ export interface ClaimKeyResponse {
 /**
  * Struct in rust
  * -----
- pub struct JsonExecutionRequest {
-    pub code: Option<String>,
-    pub ipfs_id: Option<String>,
-    pub auth_sig: AuthSigItem,
-    pub js_params: Option<serde_json::Value>,
+pub struct JsonExecutionRequest {
+  pub auth_sig: AuthSigItem,
+  #[serde(default = "default_epoch")]
+  pub epoch: u64,
+  
+  pub ipfs_id: Option<String>,
+  pub code: Option<String>,
+    pub js_params: Option<Value>,
+    pub auth_methods: Option<Vec<AuthMethod>>,
```

Comment:
Remove comments?

Reply:
no - it's helpful to see what has changed from the nodes when we are doing type conversion